lib/interferon.rb

```diff
@@ -65,10 +65,10 @@ def read_alerts
       abort("no such directory #{path} for reading alert files") \
         unless Dir.exists?(path)
 
-      Dir.glob(File.join(path, '*.rb')) do |alert_file|
+      Dir.glob(File.join(path, '**/*.rb')) do |alert_file|
         break if @request_shutdown
         begin
-          alert = Alert.new(alert_file)
+          alert = Alert.new(@alerts_repo_path, alert_file)
```

Comment:
oh oops. yeah  i guess i got relative paths because i passed a relative path to glob in my testing, but we explicitly call `expand_path` so we get absolute paths